docs/live/README.md

```diff
@@ -0,0 +1,81 @@
+## pkg
+
+Fetch, update, and sync configuration files using git
+
+<link rel="stylesheet" type="text/css" href="/kpt/gifs/asciinema-player.css" />
+<asciinema-player src="coming..." speed="1" theme="solarized-dark" cols="60" rows="26" font-size="medium" idle-time-limit="1"></asciinema-player>
+<script src="/kpt/gifs/asciinema-player.js"></script>
+
+    # run the tutorial from the cli
+    kpt tutorial live
+
+[tutorial-script]
+
+### Synopsis
+
+Tools to safely apply and remove packages from clusters.
+
+| Command   | Description                                       |
+|-----------|---------------------------------------------------|
+| [apply]   | apply a package to the cluster                    |
+| [preview] | preview the operations that apply will perform    |
+| [destroy] | remove the package from the cluster               |
+
+**Data Flow**: local configuration or stdin -> kpt live -> apiserver (Kubernetes cluster)
+
+| Configuration Read From | Configuration Written To |
+|-------------------------|--------------------------|
+| local files or stdin    | apiserver                |
+| apiserver               | stdout                   |
+
+#### Pruning
+kpt live apply handles pruning of resources. That means that 
```

Comment:
Consider the following wording for paragraph 1:

kpt live apply will automatically delete resources which have been previously applied, but which are currently omitted. This clean-up functionality is called pruning. For example, consider a package which has been applied with the following three resources:


```
service-1 (Service)
deployment-1 (Deployment)
config-map-1 (ConfigMap)
```

Then imagine the package is updated to contain the following resources, including a new ConfigMap named `config-map-2` (Notice that `config-map-1` is not part of the updated package):

```
service-1 (Service)
deployment-1 (Deployment)
config-map-2 (ConfigMap)
```

When the updated package is applied, config-map-1 is automatically deleted (pruned) since it is omitted.

Reply:
Yeah, I think using an example like this is a good idea. I have incorporated this with just a tiny adjustment.